.tool-versions

```diff
@@ -2,5 +2,4 @@
 # check it with
 # diff <(sed '/^#/d; /^$/d' .tool-versions | sort) <(sed '/^#/d; /^$/d' .tool-versions) && echo ".tool-versions is sorted correctly" || echo ".tool-versions is not sorted correctly"
 
-# renovate: datasource=github-releases depName=pre-commit/pre-commit
```

Comment:
not required, Renovate knows how to deal with `.tool-versions`. Only exception is when the asdf plugin is mismatching between download and version definition.
E.g. expected: `v1.0.0` but on GitHub it's `1.0.0`, that's where the version extract comes in.

Reply:
I'd align it with the other repositories.
If not explicitly required due to extracted versions, omit.

Reply:
I see what you mean but explicitly defining it wouldn't help you either here as long as you don't do the extract version part in renovate.
That's something to consider on a case-by-case basis when adding a new tool to adjust the renovate config to consider the `v` or not.

Infra has the following disclaimer in their `.tool-versions`, I'll add it to all of ours as well as reminder.

```
################################################################################
#                                                                              #
# Note: when adding a new tool here, also adjust the Renovate configuration    #
#       that mirrors some asdf plugin's logic to remove version prefixes like  #
#       "v".                                                                   #
#                                                                              #
#       You have to amend the Renovate config when the asdf plugin takes 1.2.3 #
#       as the version but the Docker tag/Github tag is actually v1.2.3        #
#                                                                              #
################################################################################
```